docs/advanced/1-multilevel.md

```diff
@@ -175,6 +175,122 @@ IS-IS Instance: Gandalf VRF: default
+Legend:
+H - hostname conflict
+U - node unreachable
+
+IS-IS Instance: Gandalf VRF: default
+IS-IS Level 1 Link State Database
+LSPID                   Seq Num  Cksum  Life Length IS  Received LSPID        Flags
+c1.00-00                      4  20496   392    131 L2  0000.0000.0002.00-00  <DefaultAtt>
+LSP received time: 2025-08-17 10:03:40
+Remaining lifetime received: 1199 s Modified to: 1200 s
+NLPID: 0xCC(IPv4)
+Hostname: c1
+Area addresses: 49.0100
+Interface address: 10.1.0.9
+Interface address: 10.1.0.1
+Interface address: 10.0.0.2
+Interface address: 10.1.0.13
+IS Neighbor          : c2.00               Metric: 10
+IS Neighbor          : r1.00               Metric: 10
+Reachability         : 10.1.0.8/30 Metric: 10 Type: 1 Up
+Reachability         : 10.1.0.0/30 Metric: 10 Type: 1 Up
+Reachability         : 10.0.0.2/32 Metric: 10 Type: 1 Up
+Reachability         : 10.1.0.12/30 Metric: 10 Type: 1 Up
+Router Capabilities: Router Id: 10.0.0.2 Flags: []
+Area leader priority: 250 algorithm: 0
+
+```
+
+Interestingly enough, the L1 LSP originated by C1 advertises contains the interface address of C1's Ethernet3 interface (C1-X1 link) and subsequently advertises reachability to 10.1.0.12/30 transit network, despite X1 being a level-2 only IS and C1 and X1 having an L2 adjacency relation.
+
+To understand why, we need to delve into ISIS interface circuit types.
+
+## IS-IS interface level circuit type
+
+Move to C1. Start an examination of its adjacency types and its interfaces from the point of view of IS-IS.
+
+C1's neighbors and IS-IS interface status running on Arista cEOS
+{ .code-caption }
+```
+c1#show isis neighbors
+
+Instance  VRF      System Id        Type Interface          SNPA              State Hold time   Circuit Id
+Gandalf   default  r1               L1   Ethernet1          P2P               UP    24          10
+Gandalf   default  c2               L1L2 Ethernet2          P2P               UP    27          17
+Gandalf   default  x1               L2   Ethernet3          P2P               UP    29          00
+c1#show isis interface brief
+
+IS-IS Instance: Gandalf VRF: default
+
+Interface Level IPv4 Metric IPv6 Metric Type           Adjacency
+--------- ----- ----------- ----------- -------------- ---------
+Ethernet3 L1L2           10          10 point-to-point         1
+Loopback0 L1L2           10          10 loopback       (passive)
+Ethernet1 L1L2           10          10 point-to-point         1
+Ethernet2 L1L2           10          10 point-to-point         2
+
+```
+
+At first glance, we can confirm that the adjacency relation between C1 and X1 is L2 only and established over the Ethernet3 interface.
+
+The second command shows the IS-IS circuit type (level) for all interfaces running IS-IS. Notice that the circuit type for Ethernet3 is L1L2. This has long-reaching implications. The IS-IS circuit type is configurable under interface settings and allows overriding the global is-type set in the routing process.
```

Comment:
I will do some significant editing ;) No worries, just fix the flow, and I'll take care of the rest.